lib/resources/example_json/dstu2_examples_medication_administration.rb

```diff
@@ -0,0 +1,429 @@
+module Cerner
+  module Resources
+
+    DSTU2_MEDICATION_ADMINISTRATION_BUNDLE ||= {
+      "resourceType": "Bundle",
+      "id": "a9117d85-042f-4dda-9e38-dce0286c57a4",
+      "type": "searchset",
+      "link": [
+        {
+          "relation": "self",
+          "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration?patient=4642007"
+        },
+        {
+          "relation": "next",
+          "url": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration?patient=4642007&-pageContext=H4sIAAAAAAAAADWMuwoCMRBF_yW1BJN1iU4nrqVWW9hJSCYmYB6Ms4KI_25AvN3hHs5bNHtDAeJ8vMxi1Ymw8An93udUDgvXEPq7UAGHVJAgJw8uWmKJz65CiIna3b4kY25XilX-RPto0tUMRptRb4fexuInnvM_qkal9M4Mm3Xf5wtjreoSiQAAAA%3D%3D&-pageDirection=NEXT"
+        }
+      ],
+      "entry": [
+        {
+          "fullUrl": "https://fhir-open.sandboxcernerpowerchart.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationAdministration/7283373",
+          "resource": { # REGULAR, MLTP CODINGS
+            "resourceType": "MedicationAdministration",
+            "id": "7283373",
+            "meta": {
+              "versionId": "7283374",
+              "lastUpdated": "2017-11-22T17:25:12.000Z"
+            },
+            "text": {
+              "status": "generated",
+              "div": "<div><p><b>Medication Administration</b></p><p><b>Patient Name</b>: Smart, Connie</p><p><b>Medication Display</b>: acetaminophen</p><p><b>Was Not Given</b>: False</p><p><b>Dosage Quantity</b>: 120</p><p><b>Medication Status</b>: in-progress</p><p><b>Effective Time</b>: 2017-11-22T17:25:00.000Z</p></div>"
```

Comment:
Escape the HTML in your divs. They are being rendered when viewing the site

Reply:
Escaped: https://github.com/cerner/fhir.cerner.com/pull/169/commits/a18a46531f69088dec1ee6993ef849fb497c772f#diff-e8b526c8fb4dde5a442627faec99bc0f